samtranslator/model/api/api_generator.py

```diff
@@ -634,7 +679,17 @@ def to_cloudformation(self, redeploy_restapi_parameters, route53_record_set_grou
         permissions = self._construct_authorizer_lambda_permission()
         usage_plan = self._construct_usage_plan(rest_api_stage=stage)
 
-        return rest_api, deployment, stage, permissions, domain, basepath_mapping, route53, usage_plan
+        return (
```

Comment:
nit: those long tuples are easier to maintain when they are named, no need to double-check that the order is not broken